templates/suppliers/show.html.twig

```diff
@@ -100,8 +100,9 @@
 
 
                         {% for framework in supplier.content.live_frameworks %}
+                            {% if framework.terms == "DPS" %}
```

Comment:
Is this the best way to do this? Why not invert the condition and add it to the existing one below?